appnav/src/main/kotlin/io/element/android/appnav/RoomFlowNode.kt

```diff
@@ -68,11 +70,13 @@ class RoomFlowNode @AssistedInject constructor(
             onCreate = {
                 Timber.v("OnCreate")
                 plugins<LifecycleCallback>().forEach { it.onFlowCreated(inputs.room) }
+                appUiStateService.onNavigateToRoom(inputs.room.roomId.value)
```

Comment:
If you use the RoomFlowNode, it'll include the future room details, and not just the messages screen.

Reply:
I will change. Also I can see that when the app goes to BG when this screen is displayed, the onDestroy is not called, so this is not what we expect.

Reply:
You can use `resume/pause` then

Reply:
But it will break when user will open a Thread...

Reply:
Actually, I will revert and think about another approach.